diskmaker/controllers/deleter/reconcile.go

```diff
@@ -220,8 +220,11 @@ func handlePVChange(runtimeConfig *provCommon.RuntimeConfig, pv *corev1.Persiste
 	}
 	q.Add(reconcile.Request{NamespacedName: types.NamespacedName{Namespace: ownerNamespace}})
 	if isDelete {
-		time.Sleep(time.Second * 10)
-		q.Add(reconcile.Request{NamespacedName: types.NamespacedName{Namespace: ownerNamespace}})
```

Comment:
Change itself looks good but I am doubtful about purpose of random sleep here. If I understand correctly the sleep was introduced so as cleanup status tracker inside static-provisioner has chance to mark status of PV when it starts cleaning it up. So what happens if we don't put sleep here? Is it racy? Will it cause second cleaning up of PV to be scheduled? reconciler is currently single threaded, so I am not sure how that affects.

Reply:
And why 10s? is it enough for status cleaner to catchup?

Reply:
cc @rohantmp @priyanka19-98